config/locales/notices/en.yml

```diff
@@ -26,6 +26,9 @@ en:
         access government services. We use industry-standard monitoring and
         security practices to protect your data. %{link}
       link: Learn more ›
+    use_diff_email:
+      text: Or, %{link}
```

Comment:
I usually append `_html` to translations that need to be interpreted as `html` so that happens automatically: 

http://guides.rubyonrails.org/i18n.html#using-safe-html-translations

What do you think? I think that otherwise that above will be interpreted as a string rather than html with an `a` tag, but perhaps slim is doing something fancy here that I do not understand